frontera/contrib/backends/remote/codecs/json.py

```diff
@@ -45,6 +75,10 @@ def __init__(self, request_model, *a, **kw):
         self.send_body = kw.pop('send_body', False)
         super(Encoder, self).__init__(request_model, *a, **kw)
 
+    def encode(self, obj):
+        converted = _convert(obj)
```

Comment:
This is something I haven't fixed yet. @Preetwinder Your input here would be of great value